manifests/apache.pp

```diff
@@ -59,7 +59,7 @@
   String $city = $certs::city,
   String $org = $certs::org,
   String $org_unit = $certs::org_unit,
-  String $expiration = $certs::expiration,
+  Variant[Integer, String] $expiration = $certs::expiration,
```

Comment:
I'd be open to a PR that already changes it from Strings to Integers prior to merging this.